server/middleware/articleAvailability.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-useless-return */
```

Comment:
@tuyisengepaul , do not disable Eslint flags using comments. the essence of Eslint is to enforce the Airbnb configuration. please do ensure you remove this and other comments then you resolve the Eslint error